packages/requests/index.mjs

```diff
@@ -42,9 +48,19 @@ export function startLoad (opts = {}) {
     requesters.push(wrk);
   }
 
+  if (requesters.length === 0) {
+    throw new Error('no clis available');
+  }
+
   // Start here, await in .results()
-  const toAwait = requesters.flatMap((requester) => {
-    return requests.map((request) => {
+  const toAwait = requests.flatMap((request) => {
+    // Don't allow specific requests to
+    // point at anything other than the path
+    if (request.path) {
+      request.url = request.path;
```

Comment:
Personally I'd create a new request object instead of mutating so I don't need to think as much about where this is used later. Is this because the API supports both `path` and url` as input args?

Reply:
This whole path is likely to be refactored completely when we finish support for `wrk2`. Going to leave it for now as temporary, but I agree with the goal.  Basically this was written for `autocannon` which supports both options but appends the `path` onto the `url` base. Since we hard code the `url` base since we control the server, I had some logic to merge `url` correctly but it was not accounting for `path`.

I guess now that you mention it, it would have been better to on the other side to just do `req.url || req.path`. Will change to avoid this mutation.